code/datums/components/archaeology.dm

```diff
@@ -20,27 +20,26 @@
 /datum/component/archaeology/proc/Dig(obj/item/W, mob/living/user)
 	if(dug)
 		to_chat(user, "<span class='notice'>Looks like someone has dug here already.</span>")
-		return FALSE
-	else
-		var/digging_speed
-		if (istype(W, /obj/item/shovel))
-			var/obj/item/shovel/S = W
-			digging_speed = S.digspeed
-		else if (istype(W, /obj/item/pickaxe))
-			var/obj/item/pickaxe/P = W
-			digging_speed = P.digspeed
-
-		if (digging_speed && isturf(user.loc))
-			to_chat(user, "<span class='notice'>You start digging...</span>")
-			playsound(parent, 'sound/effects/shovel_dig.ogg', 50, 1)
+		return
+		
+	var/digging_speed
+	if (istype(W, /obj/item/shovel))
```

Comment:
Oh god this file is space indented, reeee

Reply:
no it's not, I just removed some unnecessary indentation here